docs/reuse/apps/app-uninstall.md

```diff
@@ -0,0 +1,9 @@
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
```

Comment:
```suggestion
```

Reply:
@JV0812, I suddenly realized that we shouldn't add "noindex" tagging to reusable content. Once reusable content is pulled into an article, it might mean that the entire article isn't indexed! That's maybe what caused the Algolia problem. What do you think, @kimsauce?